nimare/annotate/gclda.py

```diff
@@ -14,6 +15,247 @@
 LGR = logging.getLogger(__name__)
 
 
+@jit(nopython=True, cache=True)
```

Comment:
would codecov track this function if `nopython=False`?

Reply:
I'm not sure. Let me try this out.